users/.env

```diff
@@ -19,6 +19,9 @@ KEYCLOAK_ENDPOINT = "http://dev2.badgerdoc.com/"
 KEYCLOAK_REALM = "master"
 KEYCLOAK_ROLE_ADMIN = "admin"
 
+KEYCLOAK_ROLE_ClientViewsAllUsers_ID=ClientViewsAllUsers
+KEYCLOAK_ROLE_ClientViewsAllUsers_SECRET=5ce0b47a-fc6a-4721-aba0-0b4d00134f5e
```

Comment:
Is it good idea to store SECRET in `.env` file?

Reply:
It is not and I am using it only in testing purposes.
Waiting for our DevOps engineer putting those values in 'secret' on dev environments.
It will be deleted after that